source/appModules/code.py

```diff
@@ -1,15 +1,16 @@
 # A part of NonVisual Desktop Access (NVDA)
-# Copyright (C) 2020 NV Access Limited, Leonard de Ruijter
+# Copyright (C) 2020-2024 NV Access Limited, Leonard de Ruijter, Cary-Rowen
 # This file is covered by the GNU General Public License.
 # See the file COPYING for more details.
 
 """ App module for Visual Studio Code.
 """
 
 import appModuleHandler
+import controlTypes
 from NVDAObjects.IAccessible.chromium import Document
 from NVDAObjects import NVDAObject
-
+from NVDAObjects import NVDAObjectTextInfo
```

Comment:
this should be merged with the previous line, i.e.

```suggestion
from NVDAObjects import NVDAObject, NVDAObjectTextInfo
```